app/hub/Views/RewardsView/RewardsView.jsx

```diff
@@ -43,7 +43,173 @@ class RewardsView extends Component {
+		// 		<div className="smarter-shopping-title row align-center">
+		// 			<span>A Smarter shopping experience</span>
+		// 		</div>
+		//
+		// 		<div className="smarter-shopping-description row align-center">
+		// 			<span>dkjfb dfkjsnbf kdjsnfwoih dsofihds foudshg sdoighds oija dogisjogiufsdhg ofsiuhidhfiusd iuhdsf udihsfiudh sdisduhsidsid    duhfdisuhf</span>
+		// 		</div>
+		//
+		// 		<div className="row download align-center">
+		// 			<div className="download-button">
+		// 				{t('hub_rewards_header_download')}
+		// 			</div>
+		// 		</div>
+		// 	</div>
+		//
+		// </div>
+		//
+		// <div className="ending row align-center-middle flex-dir-column">
+		// 	<div className="ending-title row">
+		// 		<span>Unique and Relevant</span>
+		// 	</div>
+		//
+		// 	<div className="ending-description row">
+		// 		<span>dkj b dkjsoa o de wgfgodd ih rfb gyfgfg gfi urioih jgdbkg dhgbsisu dmh kfd i kjdbgu su od iodhg s oihsg dhdohs osdigh d.</span>
+		// 	</div>
+		//
+		// 	<div className="ending-image row">
+		// 		<img src="/app/images/hub/rewards/group-13.svg"/>
+		// 	</div>
+		//
+		// 	<div className="ending-download row align-center align-middle">
+		// 		<div className="line columns">
+		//
+		// 		</div>
+		//
+		// 		<div className="columns download flex-container align-center-middle">
+		// 			<div className="download-button">
+		// 				{t('hub_rewards_header_download')}
+		// 			</div>
+		// 		</div>
+		//
+		// 		<div className="line columns">
+		//
+		// 		</div>
+		//
+		// 	</div>
+		// </div>
+
+		return (
+
+			<div className="background full-height">
+
+				<div className="header row align-center">
+					<div className="laptop columns">
+						<img src="/app/images/hub/rewards/ghostery-rewards-laptop.png" alt="Laptop"/>
+					</div>
+
+					<div className="header-text columns align-middle">
+
+						<span className="header-title row">
+							{t('hub_rewards_header_title')}
+						</span>
+
+						<span className="header-description row">
+							{t('hub_rewards_header_description')}
+						</span>
+
+						<div className="row learn-more">
+							<a href="https://www.ghostery.com/faqs/what-is-ghostery-rewards/" target="_blank">
+								<div className="learn-more-button">
+									{t('hub_rewards_header_learn_more')}
+								</div>
+							</a>
+						</div>
+
+					</div>
+
+				</div>
+
+				<div className="icons row align-center">
+					<div className="first-icon columns">
+
+						<div className="row align-center">
+							<img src="/app/images/hub/rewards/group-7.svg" alt="private"/>
+						</div>
+
+						<span className="first-icon-title row align-center">
+							{t('hub_rewards_first_icon_title')}
+						</span>
+
+						<span className="first-icon-description row align-center">
+							{t('hub_rewards_first_icon_description')}
+						</span>
+
+					</div>
+
+					<div className="second-icon columns">
+
+						<div className="row align-center">
+							<img src="/app/images/hub/rewards/group-6.svg" alt="smart"/>
```

Comment:
add the images using `background-image` and inline svgs in the css.  you can look at `app/scss/partials/_hub_svgs.scss`, and `app/hub/Views/SetupViews/SetupAntiSuite.scss` for an example.